products/m2e-ide/m2e-ide.product

```diff
@@ -7,8 +7,6 @@
    </configIni>
 
    <launcherArgs>
-      <programArgs>-data @noDefault
```

Comment:
@laeubi when this property is present in the launch-config the eclipse launched from the IDE asks for a workspace.
Of course I can only remove it in the launch-config, but I would like to keep them consistent.
Do you have a non-trivial reason to have it in the product?

Reply:
This program-argument has been replaced by the corresponding property that will be written to the config.ini:
`<property name="osgi.instance.area" value="@noDefault" />`
So this should behave the same for the build product but does not show-up/affect the launch-config.

Reply:
Verified with the build product, that it works.